static/manifest.json

```diff
@@ -9,6 +9,10 @@
 	"orientation": "portrait-primary",
 	"prefer_related_applications": false,
 	"icons": [
+		{
```

Comment:
why we need this?

Reply:
Improve the SEO to hopefully fix the display issue